content/docs/gke/deploy/deploy-cli.md

```diff
@@ -42,8 +42,7 @@ Follow these steps to download the kfctl binary for the Kubeflow CLI and set
 some handy environment variables:
 
 1. Download the kfctl {{% kf-latest-version %}} release from the
-  [Kubeflow releases 
-  page](https://github.com/kubeflow/kubeflow/releases/tag/{{% kf-latest-version %}}).
+  [kfctl releases page](https://github.com/kubeflow/kfctl/releases/tag/{{% kf-latest-version %}}).
```

Comment:
Thanks for fixing this URL! There are a few more places that need fixing:
https://github.com/kubeflow/website/search?q=%22https%3A%2F%2Fgithub.com%2Fkubeflow%2Fkubeflow%2Freleases%22&unscoped_q=%22https%3A%2F%2Fgithub.com%2Fkubeflow%2Fkubeflow%2Freleases%22

Reply:
Done